src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtSentencingReconciliationService.kt

```diff
@@ -255,8 +255,8 @@ class CourtSentencingReconciliationService(
       caseReferences = nomisResponse.caseInfoNumbers.map { it.reference },
     )
 
-    // TODO remove SUP/IMP hack once DPS fix their side
-    val differenceList = compareObjects(dpsFields, nomisFields).filterNot { it -> (it.nomis == "SUP" && it.dps == "IMP") }
+    // TODO remove SUP/LIC->IMP hack once DPS fix their side
+    val differenceList = compareObjects(dpsFields, nomisFields).filterNot { it -> (it.nomis == "SUP" && it.dps == "IMP") }.filterNot { it -> (it.nomis == "LIC" && it.dps == "IMP") }
```

Comment:
I could probably do this in one filter but it is a temp fix and will be removed so fine like this